pom.xml

```diff
@@ -1052,6 +1052,10 @@
     <!-- Exists for backwards compatibility; profile doesn't do anything -->
```

Comment:
So this line would be removed?

Reply:
Not necessary, this profile doesn't do anything.

Reply:
Already revert it.